ansible.cfg

```diff
@@ -0,0 +1,13 @@
+[ssh_connection]
+#pipelining=True
+ssh_args = -o ControlMaster=auto -o ControlPersist=60s -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no
+
+[defaults]
+roles_path = roles/
+inventory = inventory/
+log_path = ~/ansible.log
+private_key_file = ~/.ssh/tng-infra.pem
```

Comment:
I think Jenkins uses the default key ~/.ssh/id_rsa  instead ~/.ssh/tng-infra.pem.

Reply:
ok, to be fixed